examples/computer_vision/cifar10_pytorch/README.md

```diff
@@ -0,0 +1,29 @@
+# PyTorch CIFAR-10 CNN Example
+
+This example shows how to build a simple CNN on the CIFAR-10 dataset using
+Determined's PyTorch API. This example is adapted from this [Keras CNN
+example](https://github.com/fchollet/keras/blob/master/examples/cifar10_cnn.py).
+
+## Files
+* **model_def.py**: The core code for the model. This includes building and compiling the model.
+
+### Configuration Files
+* **const.yaml**: Train the model with constant hyperparameter values.
+* **distributed.yaml**: Same as `const.yaml`, but trains the model with multiple GPUs (distributed training).
+* **adaptive.yaml**: Perform a hyperparameter search using Determined's state-of-the-art adaptive hyperparameter tuning algorithm.
+
+## Data
+The current implementation uses CIFAR-10 data downloaded from AWS S3.
+
+## To Run
+If you have not yet installed Determined, installation instructions can be found
+under `docs/install-admin.html` or at https://docs.determined.ai/latest/index.html
+
+After configuring the settings in `const.yaml`, run the following command: `det
```

Comment:
Which settings does the user need to configure?

Reply:
Users shouldn't need to touch anything for these examples, but for their own they do. I assume that's a given so I can just say "run the following command"

Reply:
Yeah -- if there are things we do need them to configure, we should say explicitly what those are. Otherwise better to omit.